### DISCUSSION TITLE
Does Docusaurus 2.2 support relative paths?I want to generate offline static HTML files without server

### DISCUSSION THREAD

**Discussion by yangxuan21** [facebook/docusaurus#8514] [Q&A]
Does Docusaurus 2.2 support relative paths? How to use docusaurus-plugin-relative-paths plugin?I tried to install it and got the following error:

npm i docusaurus-plugin-relative-paths
npm ERR! code ENOVERSIONS
npm ERR! No versions available for docusaurus-plugin-relative-paths

info:
Docusaurus version: 2.2.0
Node version: v18.12.1

**Comment by slorber** [comment]
We don't support that feature.
See also https://github.com/facebook/docusaurus/issues/3825

docusaurus-plugin-relative-paths is (was?) not an official plugin, but demonstrated something could be done in userland so you can try to figure out how to bring support for this feature yourself until we work on it
